webpack/scenes/Subscriptions/Allocations/AllocationsPage.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Row, Col } from 'react-bootstrap';
+import { Button, Spinner, EmptyState, EmptyStateTitle, EmptyStateIcon, EmptyStateInfo, EmptyStateHelp, EmptyStateAction } from 'patternfly-react';
+
+class AllocationsPage extends Component {
+  componentDidMount() {
+    this.loadData();
+  }
+
+  loadData() {
+    this.props.loadUpstreamSubscriptions();
+  }
+
+  render() {
+    const { upstreamSubscriptions } = this.props;
+
+    const getUpstreamSubscriptionsContent = () => {
+      let table = (
+        <p>Table goes here</p>
+      );
+
+      if (upstreamSubscriptions.length === 0) {
+        table = (
+          <EmptyState>
```

Comment:
I've created a wrapper for `pf-react` empty state [here](https://github.com/theforeman/foreman/pull/5024/files#diff-8f0afc4b1c6cdbee9c65e50fe1f5c8e6R7), that renders static or dynamic data 

**Examples**
[static usage](https://github.com/theforeman/foreman/pull/5024/files#diff-079d3d1960744d6f623c1f37565fa387R16)
[dynamic usage](https://github.com/theforeman/foreman/pull/5024/files#diff-b06f92342eb07ab59365f78ad8ae760fR1)

Reply:
Nice, I will make use of that here and on subscriptions page.

Reply:
Actually, the problem with that implementation is that the empty state is supposed to be the only thing on the page per the mockup and it looks like that implementation only replaces the table component contents with the empty state.  In other words, if I used the empty state wrapper for the table I would have to manually hide all of the action buttons and the search on this page.

Mockup:
<img width="602" alt="screen shot 2018-03-13 at 8 44 49 pm" src="https://user-images.githubusercontent.com/4116405/37377307-8c0c260e-26ff-11e8-8577-697722860ec0.png">

Reply:
Upon further review it looks like this mockup isn't inline with patternfly's use of the empty state within a table view:

<img width="925" alt="screen shot 2018-03-13 at 8 50 27 pm" src="https://user-images.githubusercontent.com/4116405/37377426-3645e15a-2700-11e8-9c22-bc25de96dfd1.png">

I will make use of your component in this PR, sorry about the confusion.